src/okta/fetchBatchOfApplications.ts

```diff
@@ -0,0 +1,57 @@
+import { IntegrationLogger } from '@jupiterone/jupiter-managed-integration-sdk';
+import {
+  OktaExecutionContext,
+  OktaIntegrationStepIterationState,
+} from '../types';
+import logIfForbiddenOrNotFound from '../util/logIfForbidden';
+import retryApiCall from '../util/retryApiCall';
+import fetchBatchOfResources from './fetchBatchOfResources';
+import {
+  OktaApplication,
+  OktaApplicationCacheData,
+  OktaApplicationGroup,
+  OktaClient,
+  OktaQueryParams,
+} from './types';
+
+export default async function fetchBatchOfApplications(
```

Comment:
Same here, we won't need this code to be anything like this when `iterateApplications` exists.

Reply:
Removed